_posts/2017-01-16-Jak-psat-php-Uloziste-kodu+databaze+frameworky.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Jak psát PHP? (3/3) - Úložiště kódu, databáze a frameworky
+---
+
+V poslední části přehledu o nástrojích vhodných pro jazyk PHP se podíváme nejen na github + gitlab, řekneme si něco málo o databázích a předestřu základy frameworků. Tady můžete najít [druhý](http://jakpsatphp.cz/Jak-psat-php-Zavislosti+verzovaci-systemy+debugging/) nebo [první](http://jakpsatphp.cz/Jak-psat-php-Zdroje-znalosti+vyvojove-prostredi/) díl série. 
```

Comment:
Co ti vyhovuje. Já rád tykám, protože to tak dělám všude v reálném životě. A článek beru jako dialog se čtenářem. Málokdy je za jednímm monitorem více lidí, kteří čtou článek naráz :)